src/www/app/outline_server_repository/server.ts

```diff
@@ -29,17 +29,21 @@ export class OutlineServer implements Server {
   private static readonly SUPPORTED_CIPHERS = ['chacha20-ietf-poly1305', 'aes-128-gcm', 'aes-192-gcm', 'aes-256-gcm'];
 
   errorMessageId?: string;
-  private sessionConfig: ShadowsocksSessionConfig;
+  private sessionConfig?: ShadowsocksSessionConfig;
 
   constructor(
     public readonly id: string,
     public readonly accessKey: string,
     private _name: string,
     private tunnel: Tunnel,
     private net: NativeNetworking,
-    private eventQueue: events.EventQueue
+    private eventQueue: events.EventQueue,
+    public readonly isAccessKeyDynamic: boolean
```

Comment:
Could you change this to an access key type enum instead? I think it's clearer and removes the asymmetry.

Reply:
Also, let's add the type to the model.

Reply:
I like that idea!